altair/examples/correlation_matrix.py

```diff
@@ -0,0 +1,52 @@
+"""
+Correlation matrix
+--------------
```

Comment:
I think it's important that the length of the underline matches the length of the title when the docs are compiled in  Sphinx. You just need to add a few more dashes.